english/animals/dogs/dogs.md

```diff
@@ -0,0 +1,7 @@
+Dogs are commonly referred to as “man’s best friend”.
+
+“A dog is the only thing on earth that loves you more than he loves himself.”
+
+All you need to know about [DOGS](http://dogs.petbreeds.com/)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.